README.md

```diff
@@ -28,22 +28,18 @@ You should set up one or both of these projects with the `gcloud`
 command so that you can deploy PR versions and/or manually deploy the
 production version as necessary:
 
-    gcloud config configurations create gotraining
+    gcloud config configurations create gotraining-testing
     gcloud config set account your@email.address
-    gcloud config set project gotraining
-
-Use `gotraining-testing` instead of `gotraining` for the testing
-project.
-
-If you are on the ANZ corp network and are having SSL validation issues
-due to the proxy certs, you can disable SSL validation as a last resort:
-
-    gcloud config set auth/disable_ssl_validation
+    gcloud config set project gotraining-testing
 
 You with need to authenticate the new configuration:
 
     gcloud auth login
+    
+If you are having SSL validation issues (due to a corporate network or
+otherwise), you can disable SSL validation as a last resort:
 
+    gcloud config set auth/disable_ssl_validation
```

Comment:
I had this before the `gcloud auth login` because you may need it for the login command to work.

Reply:
Good point.

Reply:
Fair enough.